pkg/ccl/sqlccl/restore.go

```diff
@@ -213,11 +214,22 @@ func allocateTableRewrites(
 
 	// Allocate new IDs for each table.
 	//
-	// NB: we do this in a standalone transaction, not one that covers the entire
-	// restore since restarts would be terrible (and our bulk import primitive are
-	// non-transactional), but this does mean if something fails during restore
-	// we've "leaked" the IDs, in that the generator will have been incremented.
+	// NB: we do this in a standalone transaction, not one that covers the
+	// entire restore since restarts would be terrible (and our bulk import
+	// primitive are non-transactional), but this does mean if something fails
```

Comment:
This typo must be ancient, but I just noticed it: "bulk import primitive are"

Only if you catch a CI failure.